youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,589 @@
+msgid "Spanish"
+msgstr "Španělština"
+
+#: youtube_dl_gui/optionsframe.py:552
+msgid "Swedish"
+msgstr "Švédština"
+
+#: youtube_dl_gui/optionsframe.py:553
+msgid "Turkish"
+msgstr "Turečtina"
+
+#: youtube_dl_gui/optionsframe.py:569
+msgid "None"
+msgstr "Žádné"
+
+#: youtube_dl_gui/optionsframe.py:570
+msgid "Automatic subtitles (YOUTUBE ONLY)"
+msgstr "Automatické titulky (POUZE YOUTUBE)"
+
+#: youtube_dl_gui/optionsframe.py:571
+msgid "All available subtitles"
+msgstr "Všechny dostupné titulky"
+
+#: youtube_dl_gui/optionsframe.py:572
+msgid "Subtitles by language"
+msgstr "Titulky dle jazyka"
+
+#: youtube_dl_gui/optionsframe.py:578
+msgid "Subtitles"
+msgstr "Titulky"
+
+#: youtube_dl_gui/optionsframe.py:582
+msgid "Subtitles options"
+msgstr "Volby pro titulky"
+
+#: youtube_dl_gui/optionsframe.py:583
+msgid "Embed subtitles into video file (mp4 ONLY)"
+msgstr "Vložit titulky do video souboru (POUZE mp4)"
+
+#: youtube_dl_gui/optionsframe.py:585
+msgid "Playlist"
+msgstr "Seznam k přehrání"
+
+#: youtube_dl_gui/optionsframe.py:591 youtube_dl_gui/optionsframe.py:596
+msgid "Max"
+msgstr "Max"
+
+#: youtube_dl_gui/optionsframe.py:594
+msgid "Filesize"
+msgstr "Velikost souboru"
+
+#: youtube_dl_gui/optionsframe.py:599
+msgid "Min"
+msgstr "Min"
+
+#: youtube_dl_gui/optionsframe.py:728
+msgid "Retries"
+msgstr "Opakování"
+
+#: youtube_dl_gui/optionsframe.py:731
+msgid "Authentication"
+msgstr "Ověření"
+
+#: youtube_dl_gui/optionsframe.py:733
+msgid "Username"
+msgstr "Uživatelské jméno"
+
+#: youtube_dl_gui/optionsframe.py:735
+msgid "Password"
+msgstr "Heslo"
+
+#: youtube_dl_gui/optionsframe.py:737
+msgid "Video password"
+msgstr "Heslo videa"
+
+#: youtube_dl_gui/optionsframe.py:740
+msgid "Network"
+msgstr "Síť"
+
+#: youtube_dl_gui/optionsframe.py:742
+msgid "Proxy"
+msgstr "Proxy"
+
+#: youtube_dl_gui/optionsframe.py:744
+msgid "User agent"
+msgstr "Uživatelský agent"
+
+#: youtube_dl_gui/optionsframe.py:746
+msgid "Referer"
+msgstr ""
+
+#: youtube_dl_gui/optionsframe.py:749
+msgid "Logging"
+msgstr "Záznamy"
+
+#: youtube_dl_gui/optionsframe.py:751
+msgid "Enable log"
+msgstr "Povolit záznam"
+
+#: youtube_dl_gui/optionsframe.py:752
+msgid "View"
+msgstr "Zobrazit"
+
+#: youtube_dl_gui/optionsframe.py:753
+msgid "Clear"
+msgstr "Vymazat"
+
+#: youtube_dl_gui/optionsframe.py:863
+msgid "Youtube-dl command line options (e.g. --help)"
+msgstr "Volby příkazového řádku pro youtube-dl (např. --help)"
+
+#: youtube_dl_gui/optionsframe.py:866
+msgid "Extra options"
+msgstr "Další volby"
+
+#: youtube_dl_gui/optionsframe.py:868
+msgid "Debug youtube-dl"
+msgstr "Debug youtube-dl"
+
+#: youtube_dl_gui/optionsframe.py:869
+msgid "Ignore errors"
+msgstr "Ignorovat chyby"
+
+#: youtube_dl_gui/optionsframe.py:870
+msgid "Ignore youtube-dl config"
+msgstr "Ignorovat nastavení youtube-dl"
+
+#: youtube_dl_gui/optionsframe.py:871
+msgid "No mtime"
+msgstr ""
+
+#: youtube_dl_gui/optionsframe.py:872
+msgid "Prefer native HLS"
+msgstr ""
```

Comment:
Why not translate this? (https://translate.google.gr/#en/cs/Prefer)

Reply:
Translated